src/components/Settings/Notifications/NotificationsTelegram.tsx

```diff
@@ -86,6 +86,8 @@ const NotificationsTelegram: React.FC = () => {
               botUsername: values.botUsername,
             },
           });
+          mutate('/api/v1/settings/public');
```

Comment:
Why are we calling mutate here?

Reply:
The global/system Telegram notifications need to be configured in order for users to configure personal notifications, since user notifications are dependent on the bot username & bot API key.

Reply:
I don't think we need to provide any of that data into the public settings endpoint? It can just be returned with the API calls from the settings page itself.

Reply:
Oh hm, good point.  I'll change that!